gobblin-temporal/src/main/java/org/apache/gobblin/temporal/workflows/metrics/EventSubmitterContext.java

```diff
@@ -28,13 +45,13 @@
  * like original Azkaban flow name and gobblin job id
  */
 @Getter
-public class TrackingEventMetadata {
+public class EventSubmitterContext {
```

Comment:
javadoc says it's "package private"... although I agree w/ `public`.

also, please resolve `TODO: Fix this description`